IPs/RSKIP326.md

```diff
@@ -1,56 +1,96 @@
+
+#### Current signature:
 
 ```
 release_request_received(address indexed sender, bytes btcDestinationAddress, uint256 amount)
 ```
 
 The btcDestinationAddress field is currently in the bytes format.
 
-### Proposed signature:
+#### Proposed signature:
 
 ```
 release_request_received(address indexed sender, string btcDestinationAddress, uint256 amount)
 ```
 
-The btcDestinationAddress field will be updated to a user friendly string format.
+The btcDestinationAddress field will be updated to a user friendly string format (`base58` or bech32 depending on the address type).
 
-#### Data
+##### Data
 
 - sender: the RSK address of the peg-out requester.
 - btcDestinationAddress: the derived BTC address of said RSK address, where the BTC will be transferred to.
 - amount: amount in weis the user sent to the Bridge (this doesn't reflect how many BTC will be received since transfer fees will be deducted from that amount).
 
 If `RSKIP326` is activated, the event will log the `btcDestinationAddress` field in a user friendly string format.
 
+### New event, pegout_confirmed
+
+Currently no event is being emitted when a pegout that is waiting for confirmation (in the `releaseTransactionSet` structure) has the required confirmations and is moved to waiting for signatures (to the `rskTxsWaitingForSignatures`).
+Right now, to check if a pegout has enough confirmation and that is moved to the `rskTxsWaitingForSignatures` structure, we need to get the state of the bridge and look for the pegout in `releaseTransactionSet` or `rskTxsWaitingForSignatures`, which is not that performant or convenient.
+
+We need to add a new event to the Bridge that will be emitted when the Bridge moves a pegout to the `rskTxsWaitingForSignatures`.
+I propose this event is called `pegout_confirmed`.
+
+
+This event will exist after the batch pegout consensus changes (HOP, RSKIP271).
+
+```
+pegout_confirmed(bytes32 indexed btcTxHash, uint pegoutCreationBlockNumber)
+```
+
+- `btcTxHash`: the hash of the Bitcoin transaction without signatures that was just created
+- `pegoutCreationBlockNumber`: the block where the pegout transaction was created, this is just informational value and totally optional
+
+### add_signature event logging order
+
+Emit the `add_signature` event if the transaction was actually signed, not when there's an attempt to sign it.
+
 ## Backwards Compatibility
 
 This change is a hard-fork and therefore all full nodes, block explorers must be updated. SPV light-clients do not need to be updated. 
 
 ## References
 
-[1] [RSKIP-185](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP185.md) 
+[1] [RSKIP-185](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP185.md)
+[2] [RSKIP-271]https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP271.md
```

Comment:
The format is a bit odd here 
<img width="760" alt="image" src="https://user-images.githubusercontent.com/53787863/186162819-316ab8ee-9096-4c8c-a392-dbcbbc7dbb61.png">